go/lang/correctness/looppointer.go

```diff
@@ -0,0 +1,30 @@
+func() {
+    values := []string{"a", "b", "c"}
+    var funcs []func()
+    // ruleid:exported_loop_pointer
+    for _, val := range values {
+        funcs = append(funcs, func() {
+            fmt.Println(&val)
+        })
+    }
+}
+
+func() {
+    // ruleid:exported_loop_pointer
+    for _, val := range values {
+        print_pointer(val)
+    }
+}
+
+
+func() {
+    values := []string{"a", "b", "c"}
+    var funcs []func()
+    // ok
```

Comment:
Would you mind making this //ok:exported_loop_pointer (just so checks can test that this line doesn't match)